testsuite/tests/formatting/errors_batch.reference

```diff
@@ -0,0 +1,6 @@
+File "hello.ml", line 18, characters 20-27:
+Error: These are the contents of the main error message. It is very long and should wrap across several lines.
```

Comment:
Interestingly, with the changes to use `kdprintf`, this line is not wrapped anymore, and I have no idea why. Also sorry for rebasing, I just realised it would have been useful to have kept the previous version...

Reply:
Nevermind, that was a mistake on my side: a `Format.pp_print_text` was missing after refactoring `errors_batch.ml`